src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dailyCost = 40;
+
+  if (days >= 7) {
+    return days * dailyCost - 50;
+  }
+
+  if (days >= 3) {
+    return days * dailyCost - 20;
```

Comment:
Try to follow DRY (Don't repeat yourself) principe. 
You have `days * dailyCost`. Just create a variable for it